src/chatdbg/util/module_whitelist.txt

```diff
@@ -0,0 +1,72 @@
+pandas.*
+numpy.*
+scipy.*
+datascience.*
+cs104.*
+
+builtins.abs
+builtins.aiter
+builtins.all
+builtins.anext
+builtins.any
+builtins.ascii
+builtins.bin
+builtins.bool
+builtins.bytearray
+builtins.bytes
+builtins.callable
+builtins.chr
+builtins.classmethod
+builtins.complex
+builtins.delattr
+builtins.dict
+builtins.dir
+builtins.divmod
+builtins.enumerate
+builtins.filter
+builtins.float
+builtins.format
+builtins.frozenset
+builtins.getattr
+builtins.globals
+builtins.hasattr
+builtins.hash
+builtins.help
+builtins.hex
+builtins.id
+builtins.input
+builtins.int
+builtins.isinstance
+builtins.issubclass
+builtins.iter
+builtins.len
+builtins.list
+builtins.locals
+builtins.map
+builtins.max
+builtins.memoryview
+builtins.min
+builtins.next
+builtins.object
+builtins.oct
+builtins.open
+builtins.ord
+builtins.pow
+builtins.print
+builtins.property
+builtins.range
+builtins.repr
+builtins.reversed
+builtins.round
+builtins.set
+builtins.setattr
+builtins.slice
+builtins.sorted
+builtins.staticmethod
+builtins.str
+builtins.sum
+builtins.super
+builtins.tuple
+builtins.type
+builtins.vars
+builtins.zip
```

Comment:
Any reason not to `builtins.*` as above?

Reply:
eval and exec...  Fixing the broken test, etc. now.

Reply:
Makes sense.